tests/gemini_test.yaml

```diff
@@ -0,0 +1,59 @@
+test_duration: 10080
+n_db_nodes: 1
+n_loaders: 1
+n_monitor_nodes: 1
+user_prefix: 'gemini-test'
+failure_post_behavior: keep
+ip_ssh_connections: 'public'
+experimental: 'true'
+append_conf: "enable_keyspace_column_family_metrics: true"
+instance_provision: 'spot_low_price'
+# instance_provision can be one of the following: on_demand|spot_fleet|spot_low_price|spot_duration
+store_results_in_elasticsearch: False
+
+# gemini
+gemini_run: True
+# the below cmd runs about 3 hours
+gemini_cmd: "gemini -d -n 1000000 -c 20 -p 100 -m mixed -f"
+gemini_url: 'https://s3.amazonaws.com/scylla-gemini/Binaries/gemini-0.2'
```

Comment:
can we point it to gemini-latest or something like that?

Reply:
I can create a copy of the last binary on s3 and call it "latest"

Reply:
I can create on s3 a copy of the last uploaded binary and call it "latest"

Reply:
I can create on s3 copy of last uploaded binary and call it "gemini-latest"